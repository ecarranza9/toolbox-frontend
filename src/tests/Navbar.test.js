import React from 'react';
import { render } from '@testing-library/react';
import NavBar from '../components/layout/Navbar';

describe('Navbar Component', () => {
  const { asFragment } = render(<NavBar />);
  test('should match snapshot and styles', () => {
    expect(asFragment(<NavBar />)).toMatchSnapshot();
  })
});