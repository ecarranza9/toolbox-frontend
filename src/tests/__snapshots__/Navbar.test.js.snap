// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component should match snapshot and styles 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand navbar-dark bg-dark"
  >
    <div
      class="container"
    >
      <span
        class="navbar-brand"
      >
        React Test APP
      </span>
    </div>
  </nav>
</DocumentFragment>
`;
